sql/_32_damson/cbrd_22783_REGEXP_functions/answers/REGEXP_COUNT_unicode.answer

```diff
@@ -1,20 +1,22 @@
 ===================================================
 0
 ===================================================
+0
+===================================================
 regexp_count(_utf8'가나다가나다가나다라' collate utf8_ko_cs, _utf8'(가나)다' collate utf8_ko_cs, 1)    
 3     
 
 ===================================================
 regexp_count(_utf8'가나 가나다라 마바사아 자차카타 파하' collate utf8_ko_cs, _utf8'[[:alpha:]]+' collate utf8_ko_cs, 1)    
-5     
+0     
```

Comment:
@hgryoo 
The execution result seems 'regexp_engine' parameter is not set as 'cppstd'. The result is same as 're2'.